theme/migrations/0037_remove_userquota_and_userprofile_from_db.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.25 on 2024-07-15 16:36
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('theme', '0036_auto_20240702_1847_squashed_0037_remove_userprofile_create_irods_user_account'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+            ],
+            # Operations to be performed on the database.
+            database_operations=[
+                migrations.RunSQL(
+                    sql='ALTER TABLE "theme_userquota" DROP COLUMN "user_zone_value" CASCADE;',
+                    reverse_sql='ALTER TABLE "theme_userquota" ADD COLUMN "user_zone_value" varchar(255) NULL;',
```

Comment:
Just curious why we would need this rollback?

Reply:
copilot added it automatically and I didn't see any harm.

Reply:
I guess you're right, we wont use it so I will remove it